packages/astro/components/Markdown.astro

```diff
@@ -1,7 +1,16 @@
 ---
 import { renderMarkdown } from '@astrojs/markdown-support';
 
-const { content, $scope } = Astro.props;
+export interface Props {
+  content?: string;
+}
+
+// Internal props that should not be part of the external interface.
+interface InternalProps extends Props {
```

Comment:
interesting! makes sense

Reply:
Yep, exactly, so moving it to another interface means that the user won't be presented that as a prop option in prop suggestions.